src/BOL/Models/User/Role.cs

```diff
@@ -1,10 +1,11 @@
 ﻿namespace BOL.Models
 {
-    public static class Role
+    public class Role
     {
         public const string Student = "Student";
         public const string Supervisor = "Supervisor";
         public const string Administrator = "Administrator";
+        public const string Reviewer = "Reviewer";
```

Comment:
This role doesn't have a class against it. Its only a constant used for comparison instead of comparing a string.